src/coreclr/jit/simdashwintrinsiclistarm64.h

```diff
@@ -202,99 +202,99 @@ SIMD_AS_HWINTRINSIC_ID(Vector4,     WithElement,
 //                                                                                                     {TYP_BYTE,                                      TYP_UBYTE,                                      TYP_SHORT,                                      TYP_USHORT,                                     TYP_INT,                                        TYP_UINT,                                       TYP_LONG,                                       TYP_ULONG,                                      TYP_FLOAT,                                      TYP_DOUBLE}
 // *************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
 //  Vector<T> Intrinsics
-SIMD_AS_HWINTRINSIC_ID(VectorT128,  Abs,                                                    1,         {NI_VectorT128_Abs,                             NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs,                              NI_VectorT128_Abs},                             SimdAsHWIntrinsicFlag::None)
```

Comment:
the changes in this and `simdashwintrinsiclistxarch.h` are mainly renaming `VectorT128` to `VectorT`?

Reply:
Right, that's the only change here which should simplify adding future sizes to `Vector<T>` as we no longer have to duplicate table entries per size.